### PR TITLE
add recipe for moleculetda

### PR DESCRIPTION
**Pull request by kjappelbaum**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
not sure why it is trying to use py2.7 for dionysus

File: recipes/moleculetda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  commands:
+    - pip check
+    - moleculetda --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/a1k12/molecule-tda
+  summary: Toolkit for using topological data analysis representations.
+  license: BSD-3-Clause
```
Comment (dopplershift) [inline recipes/moleculetda/meta.yaml:46]: This isn't the standard BSD-3-Clause, but is:
```suggestion
  license: BSD-3-Clause-LBNL
```

File: recipes/moleculetda/LICENSE [hunk 2]
```diff
@@ -0,0 +1,42 @@
+molecule-tda Copyright (c) 2021, The Regents of the University of California,
```
Comment (dopplershift) [inline recipes/moleculetda/LICENSE:1]: The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

File: recipes/moleculetda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/moleculetda-{{ version }}.tar.gz
+  sha256: 43e2e2d21abbb1011421e98090013559c31658b350ce281ea874ec2a6c1dc1bb
+
+build:
+  entry_points:
+    - moleculetda = moleculetda.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win, py<36]
```
Comment (dopplershift) [inline recipes/moleculetda/meta.yaml:17]: This looks to be pure Python and could be noarch:
```suggestion
  noarch: python
```

I'm not sure what caused any windows failures, but if it's due to missing dependencies, we can ignore those errors.

File: recipes/moleculetda/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  host:
+    - python 
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/moleculetda/meta.yaml:24]: For noarch:
```suggestion
    - python >=3.8
    - pip
  run:
    - python >=3.8
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
A few tweaks and this should be good.

I also need confirmation from @a1k12 that they are willing to be listed as a maintainer for this recipe.

**Comment by a1k12** [comment]
I can confirm willingness to be listed as a maintainer for this.

**Comment by kjappelbaum** [comment]
@dopplershift thanks for this really constructive and useful review!

File: recipes/moleculetda/LICENSE (hunk 2 above)
Comment (kjappelbaum) [reply recipes/moleculetda/LICENSE]: Thanks, removed the license!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**